CHANGELOG.md

```
@@ -3,30 +3,34 @@
 
-# [2.12.0](https://github.com/tsparticles/editor/compare/v2.11.0...v2.12.0) (2023-08-04)
+# [3.1.0](https://github.com/tsparticles/editor/compare/v3.0.0...v3.1.0) (2024-11-25)
 
-**Note:** Version bump only for package tsparticles-editor-workspace
+### Bug Fixes
 
+- fixed issue with newer tsParticles versions ([fd97209](https://github.com/tsparticles/editor/commit/fd97209339afb6a14c0aa30b47605cf94ec4d461))
 
+# [3.0.0](https://github.com/tsparticles/editor/compare/v2.12.0...v3.0.0) (2023-12-05)
 
+### Bug Fixes
```

Markdownlint (0.35.0)

Expected: h2; Actual: h3
Heading levels should only increment by one level at a time

(MD001, heading-increment)